mirgecom/boundary.py

```diff
@@ -465,17 +477,20 @@ def viscous_divergence_flux(self, discr, btag, gas_model, state_minus,
     def _identical_grad_av(self, grad_av_minus, **kwargs):
         return grad_av_minus
 
-    def av_flux(self, discr, btag, diffusion, **kwargs):
+    def av_flux(self, discr, dd_bdry, diffusion, **kwargs):
         """Get the diffusive fluxes for the AV operator API."""
-        grad_av_minus = op.project(discr, "vol", btag, diffusion)
+        dd_bdry = as_dofdesc(dd_bdry)
+        dd_vol = dd_bdry.with_domain_tag(
+            VolumeDomainTag(dd_bdry.domain_tag.volume_tag))
```

Comment:
Clunky? :shrug: But functionally OK

Reply:
Fixed in inducer/grudge#280 with `untrace`.